scripts/lint-migration-sql.ts

```
@@ -0,0 +1,254 @@
+	name: string;
+	severity: FindingSeverity;
+	match: (line: string) => boolean;
+	message: string;
+	acknowledgeable: boolean;
+}
+
+const LINT_RULES: LintRule[] = [
+	{
+		name: "no-drop-table",
+		severity: "error",
+		match: (line) => /\bDROP\s+TABLE\b/i.test(line),
```

Biome (2.4.9)

[error] This regex literal is not defined in the top level scope. This can lead to performance issues if this function is called frequently.

(lint/performance/useTopLevelRegex)